composer.json

```diff
@@ -76,7 +76,7 @@
         "symfony/yaml": "self.version"
     },
     "require-dev": {
-        "cache/integration-tests": "^0.6",
+        "cache/integration-tests": "dev-master",
```

Comment:
We need this until the test suite fits our needs. I suggest we add a version number when the Component is stable enough/feature full, in a few weeks.